decidim-admin/config/locales/en.yml

```
@@ -114,6 +124,7 @@ en:
         show_in_footer: Show in the footer
         slug: URL slug
         title: Title
+        topic: Topic
```

This is not working for me. The field name should be `topic_id`:
```suggestion
        topic_id: Topic
```